src/ViewModels/CkanGroupViewModel.js

```diff
@@ -76,6 +77,14 @@ var CkanGroupViewModel = function(application) {
      */
     this.minimumMaxScaleDenominator = undefined;
 
+    /**
+     * Gets or sets any extra wms parameters that should be added to the wms query urls in this CKAN group.
+     * If this property is undefined or if {@link CkanGroupViewModel#filterByWmsGetCapabilities} then no parameters are added
```

Comment:
I was thinking of a different file.  Here it was meant to pass in tiled=true for an entire ckan group which is appended at line 395.  It was a quick solution for our base NM data.  It is probably not necessary when we make tiled the wms default in a later checkin.